imap_processing/cdf/config/imap_idex_l1a_variable_attrs.yaml

```diff
@@ -143,239 +142,239 @@ event_number:
 
 tof_low_trigger_num_max_2:
   <<: *trigger_base
   VALIDMAX: 65535
   FIELDNAM: TOF Low Pulse 2 Max Samples
   CATDESC: Maximum number of samples for pulse 2 for TOF Low single and double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_level:
   <<: *trigger_base
   VALIDMAX: 1023
   FIELDNAM: TOF Mid Trigger Level
   CATDESC: Trigger level for the TOF Mid Channel
   LABLAXIS: Level
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_max_1_2:
   <<: *trigger_base
   VALIDMAX: 2047
   FIELDNAM: TOF Mid Double Pulse Max Samples
   CATDESC: Maximum number of samples between pulse 1 and 2 for TOF Mid double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_min_1_2:
   <<: *trigger_base
   VALIDMAX: 2047
   FIELDNAM: TOF Mid Double Pulse Min Samples
   CATDESC: Minimum number of samples between pulse 1 and 2 for TOF Mid double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_min_1:
   <<: *trigger_base
   VALIDMAX: 255
   FIELDNAM: TOF Mid Pulse 1 Min Samples
   CATDESC: Minimum number of samples for pulse 1 for TOF Mid single and double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_max_1:
   <<: *trigger_base
   VALIDMAX: 255
   FIELDNAM: TOF Mid Pulse 1 Max Samples
   CATDESC: Maximum number of samples for pulse 1 for TOF Mid single and double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_min_2:
   <<: *trigger_base
   VALIDMAX: 255
   FIELDNAM: TOF Mid Pulse 2 Min Samples
   CATDESC: Minimum number of samples for pulse 2 for TOF Mid single and double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 tof_mid_trigger_num_max_2:
   <<: *trigger_base
   VALIDMAX: 255
   FIELDNAM: TOF Mid Pulse 2 Max Samples
   CATDESC: Maximum number of samples for pulse 2 for TOF Mid single and double pulse triggering.
   LABLAXIS: Samples Number
-  UNITS: "" # "samples" in idex_packet_parser...?
+  UNITS: " " # "samples" in idex_packet_parser...?
 
 low_sample_coincidence_mode_blocks:
   <<: *trigger_base
   VALIDMAX: 7
   FIELDNAM: LS Coincidence Mode Blocks
   CATDESC: Number of blocks coincidence window is enabled after low sample trigger.
   LABLAXIS: Blocks Number
-  UNITS: "" # "blocks" in idex_packet_parser...?
+  UNITS: " " # "blocks" in idex_packet_parser...?
```

Comment:
Is the comment still accurate or can we remove it?

Reply:
I think we can remove it! This attribute will be be removed anyway in my next PR.